packages/frontend/src/components/ProjectConnection/index.tsx

```diff
@@ -226,6 +224,7 @@ export const UpdateProjectConnection: FC<{
                 name,
                 dbtConnection,
                 warehouseConnection,
+                dbtVersion: dbtVersion ? dbtVersion : undefined,
```

Comment:
isn't this the same as`dbtVersion,` ?

Reply:
almost, if dbtVersion is "", it will flag it as undefined